go/integration_test/processor/calls_test.go

```diff
@@ -396,6 +397,66 @@ func TestProcessor_RecursiveCallsAfterAStaticCallAreStatic(t *testing.T) {
 	}
 }
 
+func TestProcessor_CallingNonExistentAccountFails(t *testing.T) {
```

Comment:
this is probably not the best name since the test is actually testing that if the account does not exists, it actually gets created.

Reply:
I agree, the name is confusing since calls to empty accounts should actually always succeed.

If I recall correctly, for such calls the empty code is assumed, which is equivalent to a code that just has a `STOP` instruction. Stops do not lead to reverts, thus the calls should succeed.